src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -68,6 +62,10 @@ def set_image
         flash[:error] = "KIWI image '#{params[:id]}' does not exist"
         redirect_back(fallback_location: root_path)
       end
+
+      def set_repositories
+        @repositories = @image.repositories.order(:order)
```

Comment:
But I would say that this is also two different conceptual things:

- The model return then ordered
- We want to show them ordered

But we can remove it... :thinking: